contracts/modules/TransactionManager.sol

```diff
@@ -214,7 +213,9 @@ abstract contract TransactionManager is BaseModule {
 
     function isAuthorised(address _wallet, address _spender, address _to, bytes calldata _data) internal view returns (bool) {
         if (_to == _spender) {
-            return authoriser.authorise(_wallet, _to, _data); // do we need to block calls to the wallet or modules?
+            return authoriser.authorise(_wallet, _to, _data); 
+            // do we need to block calls to the wallet or modules?
+            // we do need to block calls to modules for the purposes of not being able to bypass calls to startSession 
```

Comment:
If we have to do that then it will add some gas overhead to every call going through `multiCall`.

Reply:
I think it is ok to let the `Authoriser` do the authorisation and not add any constraint here. If the wallet and the module are not explicitly authorised in the `Authoriser` then the call will be blocked as expected.

However, we should probably prevent the owner to add the wallet and the module in its whitelist, i.e. add a check in `addToWhitelist`.